mkt/webapps/indexers.py

```diff
@@ -532,9 +527,18 @@ def get_app_filter(cls, request, additional_data=None, sq=None,
             should = [es_filter.Terms(id=list(set(app_ids)))]
             sq = sq[0:len(set(app_ids))]
 
+        # MUST NOT.
+        must_not = []
+        if data['profile'] and not no_filter:
```

Comment:
I'll keep `no_filter` for now: there are a few files passing this parameter and I don't want to increase the potential for making a mistake in that commit.